docs/README.md

```diff
@@ -76,6 +77,7 @@ tolerate more than a few amps, a [10A fuse to protect against short circuits](ht
 * A Raspberry Pi 3 powered by 5V regulator output, either via micro USB port or GPIO voltage pins.
 * A USB cable connecting Raspberry Pi 3 to serial bus translator board.
 * Ten serial bus servos connected in parallel to the translator board.
+* [Software source code](https://github.com/Roger-random/SGVHAK_Rover)
```

Comment:
This is great for today but FYI I have on my to-do list to write up more details on how to use SGVHAK_Rover software with Sawppy. It will be an entire software section linked from the end of this README.md

Reply:
I had added that in because I was unable to find any info on source code for the rover until I went through the JPL site and found your post saying that you had adapted the code for SGVHAK to work with SAWPPY also. I know the build was meant more for technical individuals and as a general baseline, but I also know that frustration on not being able to find needed information can kill a project for someone.